.husky/pre-commit

```diff
@@ -1,4 +1 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
-npx lint-staged
+npx lint-staged
```

Comment:
While I like conventionalising the process, I'd recommend not doing it `pre-commit`. This can be very annoying when doing rebases and temporary commits. A good place would be `pre-push` instead (test & lint).

Reply:
If you do pre push you have to manually generate the diff or run on everything. lint-staged us build to run on commit. It will stay